src/mutations/article/updateArticleState.ts

```diff
@@ -34,12 +30,7 @@ const resolver: GQLMutationResolvers['updateArticleState'] = async (
       recipientId: user.id,
     })
   }
-
-  const { userName } = user
-  publicationQueue.refreshIPNSFeed({ userName })
```

Comment:
making article from active to inactive is expected to trigger a rebuild of author's IPNS feed, any reason to remove it here?

Reply:
`refreshIPNSFeed` method now only create ipns key in db and do not actually refresh IPNS, see https://github.com/thematters/matters-server/blob/master/src/connectors/articleService.ts#L315-L341 cc @tx0c